Assets/LeapMotionModules/ElementRenderer/Scripts/Features/Sprite/LeapGuiSpriteFeature.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using UnityEngine.Rendering;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Sprites;
+#endif
+
```

Comment:
Mmmmm yeah looks like there should be some additional validation.  It's legal to have both a sprite feature and a texture feature at the same time, but not if they both share the same UV channel.